lib/Doctrine/DBAL/Driver/IBMDB2/DB2Connection.php

```diff
@@ -43,29 +26,27 @@
 
 class DB2Connection implements Connection, ServerInfoAwareConnection
 {
-    /**
-     * @var resource
-     */
-    private $_conn = null;
+    /** @var resource */
+    private $conn = null;
 
     /**
-     * @param array  $params
-     * @param string $username
-     * @param string $password
-     * @param array  $driverOptions
+     * @param mixed[] $params
+     * @param string  $username
+     * @param string  $password
+     * @param mixed[] $driverOptions
      *
-     * @throws \Doctrine\DBAL\Driver\IBMDB2\DB2Exception
+     * @throws DB2Exception
      */
     public function __construct(array $params, $username, $password, $driverOptions = [])
     {
-        $isPersistent = (isset($params['persistent']) && $params['persistent'] == true);
+        $isPersistent = (isset($params['persistent']) && $params['persistent'] === true);
```

Comment:
This can be considered a BC break. Could be replaced with `if (!empty($params['persistent']))`.